.github/workflows/bump-cli.yml

```diff
@@ -0,0 +1,38 @@
+name: Bump CLI version
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/actions/create-pr/action.yml
+  schedule:
+  - cron: 0 0 */14 * * # run every 14 days
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Bump CLI
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          scripts/replace-cli-version.sh
+      - name: Commit, Push and Open a PR
+        uses: ./.github/actions/create-pr
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Looks like the token doesn't have permission to push. I would think that `contents: write` should be enough.

Reply:
It's the right permission, but it won't work while this PR runs from a fork (by design). All write permissions get downgraded to read on forks.